src/pages/Introduce/Introduce.scss

```
@@ -0,0 +1,39 @@
+@import '../../style/variables.scss';
+@import '../../style/mixin.scss';
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap');
+
+.container {
+  @include flexCenter;
+  height: 100vh;
+}
+
+.card {
+  padding: 10px;
+  border: 1.5px solid $black-color;
+  border-radius: 10px;
+  font-family: 'Poppins', sans-serif;
+}
+
+.card-image {
+  width: 200px;
+  height: 200px;
+  background-color: $black-color;
+  background-size: cover;
+  background-position: center;
+  border-radius: 10px;
+}
+
+.name {
+  margin-top: 4px;
+  @include textStyle(14px, bold);
```

```suggestion
  @include textStyle(14px, bold);
  margin-top: 4px;
```

mixin을 가져다 쓸 때는 가장 상위에 include를 하는것이 컨벤션입니다!